File: seatmaps/v1/get/ios/SeatMaps.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

amadeus.shopping.seatMaps.get(params: ["flight-orderId": "eJzTd9f3NjIJdzUGAAp%2fAiY="],
                              onCompletion: {
                                  response, error in
                                  if error == nil {
                                      print(response!.data)
                                  }
                            })

File: seatmaps/v1/get/Node SDK/seatmaps.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Returns all the seat maps of a given order
    const response = await amadeus.shopping.seatmaps.get({
      "flight-orderId": "eJzTd9cPDPMwcooAAAtXAmE=",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
